programs/obfuscator/Obfuscator.cpp

```diff
@@ -619,7 +619,7 @@ class MarkovModel
 
 public:
     explicit MarkovModel(MarkovModelParameters params_)
-        : params(std::move(params_)), code_points(params.order, BEGIN) {}
+        : params(params_), code_points(params.order, BEGIN) {}
```

Comment:
This should not be fixed, because part or `performance-move-const-arg` related to move value only type should be disabled for us.

Reply:
ok